### ISSUE TITLE
Invalid go-sdk version during build

### ISSUE THREAD

**Issue by rhamzeh** [openfga/cli#300]
Originally posted by @shruti2522 on April 16, 2024: https://github.com/orgs/openfga/discussions/300

---

Hello everyone!!
I am trying to build openfga from source by running the `make build` command, but it is throwing the following error:
```
shruti@fedora:~/cncf/cli$ make build
==> Building binary within /home/shruti/cncf/cli/dist/fga
go: downloading github.com/openfga/go-sdk v0.3.6-0.20240313140700-3de2c059df44
internal/authorizationmodel/model.go:29:2: github.com/openfga/go-sdk@v0.3.6-0.20240313140700-3de2c059df44: invalid version: unknown revision 3de2c059df44
internal/fga/fga.go:23:2: github.com/openfga/go-sdk@v0.3.6-0.20240313140700-3de2c059df44: invalid version: unknown revision 3de2c059df44
internal/fga/fga.go:24:2: github.com/openfga/go-sdk@v0.3.6-0.20240313140700-3de2c059df44: invalid version: unknown revision 3de2c059df44
make: *** [Makefile:59: build] Error 1

```
Can anyone help me resolve this?

**Comment by rhamzeh** [comment]
@shruti2522 I'm not able to reproduce, can you let me know what commit you are on?

**Comment by shruti2522** [comment]
> @shruti2522 I'm not able to reproduce, can you let me know what commit you are on?

Sure. Here's the git log.
```
shruti@fedora:~/cncf/cli$ git log
commit f62751c05f31fa7c8c01070d9bcbb001b4f29e77 (HEAD -> main, origin/main, origin/HEAD)
Merge: cba397d 0aa69e8
Author: Raghd Hamzeh <raghd@rhamzeh.com>
Date:   Wed Apr 10 03:42:12 2024 +0000

    chore(deps): bump github.com/openfga/openfga from 1.5.1 to 1.5.2 in the dependencies group (#294)

```

**Comment by ewanharris** [comment]
@shruti2522, we just merged #301 which pulls in a new version of the go-sdk, would you mind pulling the latest from `main` and see if it works for you now?

**Comment by shruti2522** [comment]
Hey @ewanharris, thanks for the heads up! I pulled the latest changes from main, and everything's running smoothly now.🎉
